kahuna/public/js/components/gr-chips/gr-chips.css

```diff
@@ -0,0 +1,64 @@
+gr-chips {
+    white-space: nowrap;
+    overflow: hidden;
+}
+
+/* Make sure the last text token is wide enough */
+gr-chips .gr-chips__chip-wrapper:last-child gr-text-chip input {
+    min-width: 2em;
+}
+
+.gr-chips__placeholder {
+    position: absolute;
+    color: #aaa;
+    -webkit-user-select: none;
```

Comment:
I checked and there is a draft and [good browser support](http://caniuse.com/#search=user-select) so I think it should be OK.